installer-static.tpl.sh

```diff
@@ -0,0 +1,110 @@
+
+  echo "deb https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/{{ rtype }}/series-{{ tarantool_version }}/linux-deb/ static main" > ${apt_source_path}
+  echo "deb https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/release/modules/linux-deb static main" >> ${apt_source_path}
+
+  mkdir -p /etc/apt/preferences.d/
+  echo -e "Package: tarantool\nPin: origin download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}\nPin-Priority: 1001" > /etc/apt/preferences.d/tarantool
+  echo -e "\nPackage: tarantool-common\nPin: origin download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}\nPin-Priority: 1001" >> /etc/apt/preferences.d/tarantool
+  echo -e "\nPackage: tarantool-dev\nPin: origin download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}\nPin-Priority: 1001" >> /etc/apt/preferences.d/tarantool
+  echo "The repository is setup! Tarantool can now be installed."
+
+  apt-get update
+
+  echo
+  echo "Tarantool {{ tarantool_version }} is ready to be installed by 'apt-get -y install tarantool'"
+
+  if [[ ${FORCE_INSTALL_TARANTOOL:-False} = "True" ]]; then
+    DEBIAN_FRONTEND=noninteractive apt-get -y install tarantool
+  fi
+}
+
+install_yum_dnf ()
+{
+  source_enabled="0"
+  cat <<EOF > /etc/yum.repos.d/tarantool_{{ tarantool_version }}.repo
+[tarantool]
+name=Tarantool
+baseurl=https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/{{ rtype }}/series-{{ tarantool_version }}/linux-rpm/static/\$basearch/
+gpgkey=https://download.tarantool.org/tarantool/{{ rtype }}/series-{{ tarantool_version }}/gpgkey
+repo_gpgcheck=1
+gpgcheck=0
+enabled=1
+priority=1
+
+[tarantool_modules]
+name=Tarantool modules
+baseurl=https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/release/modules/linux-rpm/static/\$basearch/
+gpgkey=https://download.tarantool.org/tarantool/release/modules/gpgkey
+repo_gpgcheck=1
+gpgcheck=0
+enabled=1
+priority=1
+EOF
+
+  if yum --version > /dev/null 2>&1; then
+    if [[ ${FORCE_INSTALL_TARANTOOL:-False} = "True" ]]; then
+      yum -y install tarantool
+    else
+      echo "Tarantool is ready to be installed by 'yum install -y tarantool'"
+    fi
+  else
+    if [[ ${FORCE_INSTALL_TARANTOOL:-False} = "True" ]]; then
+      dnf -y install tarantool
+    else
+      echo "Tarantool {{ tarantool_version }} is ready to be installed by 'dnf install -y tarantool'"
+    fi
+  fi
+}
+
+install_zypper ()
+{
+  zypper ar -f https://download.tarantool.org{% if usr_id %}/{{ usr_id }}{% endif %}/tarantool/{{ rtype }}/series-{{ tarantool_version }}/linux-rpm/static/${ARCH}/ tarantool
+  zypper ar -f https://download.tarantool.org/tarantool/release/modules/linux-rpm/static/${ARCH}/ tarantool_modules
+  rpm --import https://download.tarantool.org/tarantool/{{ rtype }}/series-{{ tarantool_version }}/gpgkey
+  rpm --import https://download.tarantool.org/tarantool/release/modules/gpgkey
+  zypper refresh
```

Comment:
Doesn't `zypper` do it automatically when a user executes `zypper install -y tarantool`? Looks like unneeded step.

Reply:
OK